source/localizable/feedspec/index.el.html.md.erb

```diff
@@ -180,6 +180,7 @@ ImageBot v1".
 [InStock](#instock)                                    | List    | <span class="mono">-</span>    | Όχι         | `Y`
 [Shipping Costs](#shipping-costs)                      | Decimal | <span class="mono">-</span>    | Όχι         | `3.22`
 [Color](#color)                                        | String  | <span class="mono">100</span>  | Όχι***      | `Μαύρο`
+[Description](#description)                            | String  | <span class="mono">1000</span> | Όχι         | `Σετ 3 τρόλεϊ.Οι βαλίτσες διαθέτουν 4 διπλές ρόδες και κλειδαριά με συνδυασμό. Με δυο ανατομικές λαβές, μια στο επάνω μέρος και μια στο πλάι. Διαθέτουν εσωτερικό χώρισμα και θήκες οργάνωσης. Διαστάσεις:70 cm: 48 x 70 x 28 cm. Βάρος: 4,8 Kg. Λίτρα: 94lt60 cm: 41 x 58 x 25 cm. Βάρος: 4,1 Kg. Λίτρα: 60lt. Καμπίνας: 35 x 55 x 20 cm. Βάρος: 2,7 Kg. Λίτρα: 38lt Υλικό: Light weight ABS και μηχανισμός αλουμινίου.`
```

Comment:
Add space after `Σετ 3 τρόλεϊ.`
Add space after `:` in `Διαστάσεις:70 cm`

If I am not wrong, this example studies a luggage set of 3 items: `Διαστάσεις` requires a mental parse first in order to understand it. So, I propose to make it clearer when talking for every item separately.

```
Βαλίτσα 70 cm: Διαστάσεις 48 x 70 x 28 cm, Βάρος 4,8 Kg, Χωρητικότητα 94lt. Βαλίτσα 60 cm: Διαστάσεις 41 x 58 x 25 cm, Βάρος 4,1 Kg, Χωρητικότητα 60lt. Βαλίτσα Καμπίνας: Διαστάσεις 35 x 55 x 20 cm, Βάρος 2,7 Kg, Χωρητικότητα 38lt.
```
Apart from adding `Βαλίτσα 70 cm`, `Βαλίτσα 60 cm`, `Βαλίτσα Καμπίνας`, I changed  `Λίτρα` with `Χωρητικότητα`. Additionally, I changed dots with commas between specs for every case. I used dots only for separating the 3 cases.